packages/forms-web-app/src/validators/full-appeal/contact-details.js

```diff
@@ -25,6 +25,7 @@ function validateEmail(email) {
   throw new Error('Enter an email address in the correct format, like name@example.com');
 }
 
+// This rule was used for the email field on the page
```

Comment:
remove rule?

Reply:
No as it is needed to validate the email coming from the new field. It is just not needed for the current PR.

Reply:
Will move it to appropriate file once the new page has been created